packages/repository/src/model.ts

```diff
@@ -137,6 +137,32 @@ export class ModelDefinition {
   }
 }
 
+function asJSON(value: any): any {
+  if (value == null) return value;
```

Comment:
Should there be a similar check for `undefined`?

Reply:
Please note `== null` is `true` for both `undefined` and `null`.